src/Gateway/PayplugGateway.php

```diff
@@ -541,6 +541,46 @@ public function init_form_fields()
 				'desc_tip'    => false,
 				'type' => 'oney_product_animation'
 			],
+			'satispay'        => [
```

Comment:
you can remove all of this, this is not used with the new bo!

also should be removed all the language variables that are not used